javaslang/src/test/java/javaslang/AbstractValueTest.java

```diff
@@ -269,8 +276,8 @@ public void shouldConvertToHashMap() {
         //Check order - order is matter
```

Comment:
Oh my gosh - I've overseen that. A HashMap has no order by definition. Will fix the tests.